django/db/backends/base/creation.py

```diff
@@ -174,7 +174,7 @@ def _create_test_db(self, verbosity, autoclobber, keepdb=False):
                     try:
                         if verbosity >= 1:
                             print("Destroying old test database '%s'..."
-                                  % self.connection.alias)
+                                  % test_database_name)
```

Comment:
Compare with [L49](https://github.com/scop/django/blob/testdb-destroy/django/db/backends/base/creation.py#L49) and [L207](https://github.com/scop/django/blob/testdb-destroy/django/db/backends/base/creation.py#L207). I think we should aim to do the same everywhere. Perhaps even factor out a `_database_display_str()` method.